ibm_catalog.json

```diff
@@ -406,6 +406,10 @@
             {
               "key": "project_url",
               "description": "URL of the Watson project to create"
+            },
+            {
+              "key": "resource_group_id",
+              "description": "ID of the resource group used to provision the resources"
```

Comment:
You don't need to duplicate every input variable here. I suggest you remove them so you don't need to maintain them in 2 places. By default catalog will use the default variables that is in the terraform files. You only need to add any here that you may wish to add further customisation to (such as setting a region picker dropdown etc)

Reply:
done